### ISSUE TITLE
Document s3 permissions

### ISSUE THREAD

**Issue by joe-elliott** [grafana/tempo#363]
**Is your feature request related to a problem? Please describe.**
We currently support s3, but don't have a list of the required permissions.  Add those.

**Comment by mdisibio** [comment]
From testing failure handling here https://github.com/grafana/tempo/issues/59,  have determined the minimal IAM policy.  

```json
{
    "Version": "2012-10-17",
    "Statement": [
        {
            "Sid": "TempoPermissions",
            "Effect": "Allow",
            "Action": [
                "s3:PutObject",
                "s3:GetObject",
                "s3:ListBucket",
                "s3:DeleteObject"
            ],
            "Resource": [
                "arn:aws:s3:::<bucketname>/*",
                "arn:aws:s3:::<bucketname>"
            ]
        }
    ]
}
```

**Comment by annanay25** [comment]
Nice @mdisibio!

**Comment by joe-elliott** [comment]
Fixed by #367
